packages/itwin/grouping-mapping-widget/src/widget/components/GroupingMapping.scss

```diff
@@ -5,6 +5,9 @@
 @import "~@itwin/itwinui-css/scss/variables";
 
 .gmw-group-mapping-container {
+  position: absolute;
```

Comment:
This looks like a hack. How does this solve a "text overflow" issue? And what was the text overflow issue?

Reply:
The viewers core widget container has no set width so it doesn't constrain the widget to trigger `text-overflow: ellipsis`. The hackier method is to define a `min-width: 0;` in the `nz-content` class that is a part of the viewer which is what we don't want to do. Or do this which is to position the widget itself relative to the container `nz-widget-contentContainer` and have it resize accordingly. 

Before: 
![image](https://user-images.githubusercontent.com/26608199/192853115-3677769b-f868-468b-8800-93d3ac1c44d0.png)
After:
![image](https://user-images.githubusercontent.com/26608199/192853161-0f0ea220-29b9-47cf-8004-124702b7e1fc.png)